--- libcontainerd/remote_linux.go.orig	2016-05-04 19:41:32.000000000 +0900
+++ libcontainerd/remote_linux.go	2016-05-10 23:02:24.278869120 +0900
@@ -349,9 +349,9 @@
 	}
 
 	// Start a new instance
-	args := []string{"-l", r.rpcAddr, "--runtime", "docker-runc"}
+	args := []string{"-l", "unix://" + r.rpcAddr, "--runtime", "docker-runc", "--shim", "docker-containerd-shim", "--metrics-interval=0"}
 	if r.debugLog {
-		args = append(args, "--debug", "--metrics-interval=0")
+		args = append(args, "--debug")
 	}
 	if len(r.runtimeArgs) > 0 {
 		for _, v := range r.runtimeArgs {
